apps/dashboard/lib/zod-helper.ts

```diff
@@ -44,6 +44,20 @@ export const formSchema = z.object({
             .int()
             .min(1)
             .positive(),
+            refillDay: z.coerce
+            .number({
+              errorMap: (issue, { defaultError }) => ({
+                message:
+                  issue.code === "invalid_type"
+                    ? "Refill day must be greater than 0 and a integer 31 or less"
```

Comment:
there was a suggestion by the bot above, how to word this slightly better.

Reply:
Changed wording.